packages/expo-updates/e2e/__tests__/fixtures/App-apitest.tsx

```diff
@@ -100,20 +106,22 @@ const useUpdateEvents = (listener) => {
  * @param {*} timeout Timeout in ms
  * @returns a Promise that resolves after the timeout has elapsed
  */
-const delay = (timeout) => {
+const delay = (timeout: number) => {
   return new Promise((resolve) => {
     setTimeout(resolve, timeout);
   });
 };
 
-const manifestToString = (manifest) => {
-  return JSON.stringify(
-    {
-      id: manifest.id,
-      createdAt: manifest.createdAt,
-      metadata: manifest.metadata,
-    },
-    null,
-    2
-  );
+const manifestToString = (manifest: Updates.Manifest | undefined) => {
```

Comment:
```suggestion
const manifestToString = (manifest?: Updates.Manifest) => {
```

Ypes simplification.

Reply:
Fixed